src/parsers/dr_ste_v1_tag_parser.py

```diff
@@ -417,8 +417,7 @@ def mlx5_ifc_ste_def0_v1_bits_parser(bin_str):
     ret = {}
     ret["metadata_reg_c_0"] = _val(bin_str[0: 32])
     ret["metadata_reg_c_1"] = _val(bin_str[32: 64])
-    ret["dmac_47_16"] = _val(bin_str[64: 96])
-    ret["dmac_15_0"] = _val(bin_str[96: 112])
+    ret["dmac"] = _val(bin_str[64: 96] + bin_str[96: 112])
```

Comment:
maybe fix other similar smacs and dmacs?
and shorted it to 
`ret["dmac"] = _val(bin_str[64:  112])`